source/ranges.tex

```diff
@@ -6249,7 +6257,7 @@
 \pnum
 \effects
 Initializes \exposid{base_} with \tcode{views::all(std::forward<R>(r))}, and
-\exposid{pattern_} with \tcode{single_view\{std::move(e)\}}.
+\exposid{pattern_} with \tcode{views::\linebreak single(std::move(e))}.
```

Comment:
@timsong-cpp: is this what you had in mind?

Reply:
Yes, thanks!

Reply:
Thanks for pointing it out!